tjhsst/templates/labs/list.html

```diff
@@ -8,7 +8,9 @@
 
 {% block main %}
     <div>{{ labs|length }} lab{{ labs|length|pluralize:",s" }}</div>
-
+    <div><a href="find">
```

Comment:
We definitely need a link to `/labs/find` somewhere, but I see a few problems here:

1. Hardcoding URLs is bad practice in Django. If you use `<a href="{% url 'labs:find' %}">` instead, then it will always use the correct URL even if we change it in urls.py.
2. Don't put the `<h5>` inside the `<a>`, put the `<a>` inside the `<h5>`. Block elements (like `<h1>`, `<h2>`, etc.) take up the full width of their container, so putting a link around them not only makes their contents (here, the text "Find your labs!") into a link, it makes that entire line (a strip extending to the right edge of the screen) clickable.
3. I think there are certain contexts in which this template is used (search page?) in which an argument could be made for excluding the link on that specific page. I'm not saying you should change it now, but that's something we should discuss in-person with the entire team.